modules/dr-restoring-cluster-state.adoc

```diff
@@ -53,7 +53,7 @@ This procedure assumes that you copied the `backup` directory containing the etc
 +
 [NOTE]
 ====
-It is not required to manually stop the pods on the recovery host. The recovery script will stop the pods on the recovery host.
+Stop the static pods on the control plane nodes other than the recovery host.
```

Comment:
At this point the admonition is worded very similarly to the step it modifies. I would consider either of the following alternatives:

1. Replace line 52 with this sentence and get rid of this admonition.
2. Reword this note to something like the following:
```suggestion
You do not need to stop the static pods on the recovery host.
```